lib/spack/docs/getting_started.rst

```diff
@@ -414,7 +414,17 @@ provides no Fortran compilers.  The user is therefore forced to use a
 mixed toolchain: XCode-provided Clang for C/C++ and GNU ``gfortran`` for
 Fortran.
 
-In the simplest case, you can just edit ``compilers.yaml``:
+#. First, you need to make sure that command-line tools are installed. To that
+   end run ``$ xcode-select --install``.
+
+#. Next you need to run ``$ spack compiler find`` to locate Clang.
+
```

Comment:
The words "First" and "Next" aren't need above; they are implied by the numbered list.